src/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+  <script src="https://kit.fontawesome.com/f836ddcdbf.js" crossorigin="anonymous"></script>
+	<title>To Do List</title>
+</head>
+<body>
+	<section class="section">
+        <div class="container">
+            <div class="heading">
+                <h3 class="list-header">Today's To Do</h3>
+                <span><i class="fa-solid fa-rotate"></i></span>
+            </div>
+            <div class="input-items">
+                <input id="inputs" class="input-text" type="text" placeholder="Add to your list...">
+                <span><i id="enter-key" class="fa-solid fa-right-to-bracket"></i></span>npm run stat
+            </div>
```

Comment:
- [ **OPTIONAL** ] I think it would be better if you remove the `npm run stat` text from your code base to make your UI clean and smart.